spec/models/user_authorization_document_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe UserAuthorizationDocument do
+  describe "validations" do
+    describe "attachment validation" do
+      let(:user) { create(:user) }
+      let(:user_authorization_document) do
+        described_class.new(
+          authable: user,
+          category: category,
+          attachment: attachment,
+          expires_at: Date.current
+        )
+      end
+
+      subject { user_authorization_document }
+
+      context "when the attachment url is present" do
```

Comment:
Isn't easier to read if you first have a `required` case and then the `validation` ones? It could go like:

```Ruby
context "when the category's document requires an attachment" do
	context "and the attachment is present" do
	end

	context "and the attachment is empty" do
	end
end

context "when the category's document does not require the attachment" do
	context "and the attachment is present" do
	end
	
	context "and the attachment is empty" do
	end
end
```